grblas/tests/test_matrix.py

```diff
@@ -399,6 +522,74 @@ def test_assign_column(A, v):
     assert C.isequal(result)
 
 
+def test_assign_row_scalar(A, v):
+    C = A.dup()
+    C[0, :](v.S) << v
+    D = A.dup()
+    D(v.S)[0, :] << v
```

Comment:
Are we not changing this behavior? I know this is how the GraphBLAS spec is written, but with subassign, this now feels wrong to be able to write this.

Reply:
After discussing, we will keep the original GraphBLAS spec behavior and also add a recipe for using a mask which matches the size of D as an alternate style for this call.